recipes-ni/cgroups-v2/files/createcpuacctgroups.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
```

Comment:
I think we can drop the `createcpuacctgroups.sh` script entirely. With cgroups v2 the cpuacct is not a separate cgroup controller. We can get CPU utilization stats directly from cpu.stat which is available with cpuset.

As it stands this just ends up running the same `setup_mount_and_chown` and `assign_tasks_from_parent` like the other `createcpusets.sh` script.

Sorry, I didn't catch this on the first pass.

Reply:
Fixed.